lib/fathead/mongoosejs/output.txt

```diff
@@ -0,0 +1,590 @@
+Schema-remove	A			Schema						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Schema#remove(path)</code></pre><p>Removes the given path (or [paths]).</p><span class="prog__sub">Parameters:</span><ul><li><p><code>path</code> <code>String, Array</code> </p></li></ul></section>	http://mongoosejs.com/docs/api.html#schema_Schema-remove
+Schema-requiredPaths	A			Schema						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Schema#requiredPaths(invalidate)</code></pre><p>Returns an Array of path strings that are required by this schema.</p><span class="prog__sub">Parameters:</span><ul><li><p><code>invalidate</code> <code>Boolean</code> refresh the cache</p></li></ul><span class="prog__sub">Returns:</span><ul><li><p><code></code> <code>Array</code> </p></li></ul></section>	http://mongoosejs.com/docs/api.html#schema_Schema-requiredPaths
+=Schema	A			Schema						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Schema(definition, [options])</code></pre><p>Schema constructor.</p><span class="prog__sub">Events:</span><ul><li><p><code>init</code>: Emitted after the schema is compiled into a <code>Model</code>.</p></li></ul><span class="prog__sub">Parameters:</span><ul><li><p><code>definition</code> <code>Object</code> </p></li><li><p><code>[options]</code> <code>Object</code> </p></li></ul><span class="prog__sub">Example:</span><pre><code>var child = new Schema({ name: String });\nvar schema = new Schema({ name: String, age: Number, children: [child] });\nvar Tree = mongoose.model('Tree', schema);\n\n// setting schema options\nnew Schema({ name: String }, { _id: false, autoIndex: false })</code></pre><span class="prog__sub">Options:</span>\n<li>autoIndex: bool - defaults to null (which means use the connection&apos;s autoIndex option)</li>\n<li>bufferCommands: bool - defaults to true</li>\n<li>capped: bool - defaults to false</li>\n<li>collection: string - no default</li>\n<li>emitIndexErrors: bool - defaults to false.</li>\n<li>id: bool - defaults to true</li>\n<li>_id: bool - defaults to true</li>\n<li><code>minimize</code>: bool - controls document#toObject behavior when called manually - defaults to true</li>\n<li>read: string</li>\n<li>safe: bool - defaults to true.</li>\n<li>shardKey: bool - defaults to <code>null</code></li>\n<li>strict: bool - defaults to true</li>\n<li>toJSON - object - no default</li>\n<li>toObject - object - no default</li>\n<li>typeKey - string - defaults to &apos;type&apos;</li>\n<li>useNestedStrict - boolean - defaults to false</li>\n<li>validateBeforeSave - bool - defaults to <code>true</code></li>\n<li>versionKey: string - defaults to &quot;__v&quot;</li>\n<span class="prog__sub">Note:</span><p>When nesting schemas, (children in the example above), always declare the child schema first before passing it into its parent.</p></section>	http://mongoosejs.com/docs/api.html#schema_Schema
```

Comment:
@Lord-Lavios looks like somehow we're producing a list of `<li>` here that aren't wrapped in a `<ul>`?

https://beta.duckduckgo.com/?q=%3DSchema&ia=about&iax=1

Please ensure the `<li>` are wrapped and `<ul>` that we want to show bullets have `class="prog__ul"`